src/app/api/delegates/getDelegates.ts

```diff
@@ -62,19 +62,23 @@ export async function getDelegates({
     pageSize
   );
 
-  const statements = await Promise.all(
-    delegates.map((delegate) =>
-      getDelegateStatement({ addressOrENSName: delegate.delegate })
-    )
+  const _delegates = await Promise.all(
+    delegates.map(async (delegate) => {
+      return {
+        citizen: await isCitizen(delegate.delegate),
+        statement: await getDelegateStatement({ addressOrENSName: delegate.delegate })
+      }
+    })
   );
 
   return {
     meta,
     delegates: delegates.map((delegate, index) => ({
       address: delegate.delegate,
       votingPower: delegate.voting_power?.toFixed(0),
-      statement: statements[index],
-    })),
+      citizen: _delegates[index].citizen.length > 0,
```

Comment:
Why this and not boolean check?

Reply:
Gotcha. I think this is fine, something has me worried but we might want to do something with that metadata.